src/python/visclaw/data.py

```diff
@@ -289,7 +291,7 @@ def getgauge(self, gaugeno, outdir=None, verbose=True):
                     print "continuing..."
                     return None
                     
-                gauge_data = amrclaw.GaugeData()
+                gauge_data = amrclaw.GaugeData(self.num_dim)
                 gauge_data.read(outdir)
 
```

Comment:
Not sure how to get arorund using num_dim here (in order to pass this info along to GaugeData in amrclaw.